src/os/filestore/FileJournal.cc

```diff
@@ -189,39 +189,37 @@ int FileJournal::_open_file(int64_t oldsize, blksize_t blksize,
   }
 
   if (create && (oldsize < conf_journal_sz)) {
-    uint64_t newsize(cct->_conf->osd_journal_size);
-    newsize <<= 20;
-    dout(10) << "_open extending to " << newsize << " bytes" << dendl;
-    ret = ::ftruncate(fd, newsize);
+    dout(10) << "_open extending to " << conf_journal_sz << " bytes" << dendl;
```

Comment:
1) yes, __func__ is useful for a log info, I will create a new commit for it in same pr, since it is not related to the first topic.

2) agree. The purpose of my commit is: I just hate to <<=20 twice for same value ...